src/digid_eherkenning_oidc_generics/mixins.py

```diff
@@ -31,3 +48,58 @@ class SoloConfigEHerkenningMixin(SoloConfigMixin):
 class SoloConfigDigiDMachtigenMixin(SoloConfigMixin):
     config_class = OpenIDConnectDigiDMachtigenConfig
     settings_attribute = digid_machtigen_settings
+
+
+class SoloConfigEHerkenningBewindvoeringMixin(SoloConfigMixin):
+    config_class = OpenIDConnectEHerkenningBewindvoeringConfig
+    settings_attribute = eherkenning_bewindvoering_settings
+
+
+class MachtigenBackendMixin:
+    def get_or_create_user(self, access_token, id_token, payload):
+        claims_verified = self.verify_claims(payload)
+        if not claims_verified:
+            msg = "Claims verification failed"
+            raise SuspiciousOperation(msg)
+
+        self.extract_claims(payload)
+
+        user = AnonymousUser()
+        user.is_active = True
+        return user
+
+    def extract_claims(self, payload: dict) -> None:
+        self.request.session[self.session_key] = {}
+        for claim_name in self.claim_names:
+            self.request.session[self.session_key][claim_name] = glom(
+                payload, claim_name
+            )
+
+    def log_received_claims(self, claims: dict):
+        copied_claims = deepcopy(claims)
+
+        def _obfuscate_claims_values(claims_to_obfuscate: dict) -> dict:
+            for key, value in claims_to_obfuscate.items():
+                if isinstance(value, dict):
+                    _obfuscate_claims_values(value)
+                else:
+                    claims_to_obfuscate[key] = obfuscate_claim(value)
+            return claims_to_obfuscate
```

Comment:
Could we use the `mozilla_django_oidc_db.utils.obfuscate_claims` util here that handles this recursion?

Reply:
The `mozilla_django_oidc_db.utils.obfuscate_claims` requires to give a list of which claims should be obfuscated, while I kind of when for 'obfuscate everything' since the information that comes back in the token depends on the OIDC provider and would have to be configured?

Reply:
ok